platforms/linux/org.qlcplus.QLCPlus.appdata.xml

```diff
@@ -67,15 +67,15 @@
  </description>
  <screenshots>
   <screenshot type="default">
-   <image>http://www.qlcplus.org/shots/qlcplus_fixtures.png</image>
+   <image>https://www.qlcplus.org/user/pages/01.home/Fixtures.PNG</image>
```

Comment:
This one I uploaded this
https://www.qlcplus.org/user/pages/05.discover/01.features/02._fixtures/001-fixtures-2.png